common/scripts/x86_64/CentOS_7/_installVersion.env

```diff
@@ -0,0 +1,2 @@
+readonly DOCKER_VERSION=17.06
+readonly AWSCLI_VERSION=1.11.91
```

Comment:
Same version as that of Ubuntu 16.04. This is the version [where `--no-include-email` support](https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst#11191) was added.